LiskHQ/lisk-hub
# Implement wallet on-boarding - Closes #2163

### What issue have I solved?

#2163

### How have I implemented/fixed it?

- [x] Created onboarding using the same component we use in delegates and dashboard
- [x] Remove old wallet "New account" banner. I'm not sure if this was expected, it's not part of acceptance criteria in #2163, but it's also not in the designs https://projects.invisionapp.com/d/main#/console/17570736/365248394/preview and it looks weird with both. 
- I didn't put the dots at the end of headlines (e.g. "Manage your LSK with ease") even though they are in the designs because delegates and dashboard also don't use them.

### How has this been tested?

- Log in
- Go to wallet
- See the onboarding, click through it, close it...

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
